pkg/stanza/fileconsumer/finder_test.go

```diff
@@ -243,23 +291,35 @@ func TestFinder(t *testing.T) {
 			files:   []string{"err.a.123456786.2023020610.log", "err.b.123456789.2023020609.log", "err.c.123456788.2023020612.log", "err.d.123456787.2023020611.log"},
 			include: []string{"err.*.log"},
 			exclude: []string{},
-			filterSortRule: []SortRule{
-				{
-					Regex:     `err\.[a-zA-Z]\.(?P<value>\d+)\.\d{10}\.log`,
-					SortType:  SortTypeNumeric,
-					Ascending: false,
-				},
-				{
-					Regex:     `err\.[a-zA-Z]\.\d+\.(?P<value>\d{10})\.log`,
-					SortType:  SortTypeTimestamp,
-					Ascending: false,
-					Location:  "UTC",
-					Format:    `%Y%m%d%H`,
-				},
-				{
-					Regex:     `err\.(?P<value>[a-zA-Z])\.\d+\.\d{10}\.log`,
-					SortType:  SortTypeAlphabetical,
-					Ascending: false,
+			filterSortRule: OrderingCriteria{
+				Regex: `err\.(?P<alpha>[a-zA-Z])\.(?P<number>\d+)\.(?P<time>\d{10})\.log`,
+				SortBy: []SortRuleImpl{
+					{
+						NumericSortRule{
+							BaseSortRule: BaseSortRule{
+								RegexKey:  "number",
+								Ascending: false,
```

Comment:
Can we use `Ascending: true` in one or both of these multi-step sort cases?

Reply:
Updated.